### ISSUE TITLE
Make Windows agents to send the keep-alive independently

### ISSUE THREAD

**Issue by chemamartinez** [wazuh/wazuh#3963]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.10.0-31008 | N/A | Agent | N/A | Windows |

### Bug description

The trigger to send a keepalive message from the Windows agent, is the sending of a new event to the manager. Every time a new event is generated, the agent checks if the _notify time_ has been reached, if so, the keepalive is sent with the event.
That provokes the agent not to send any keep-alive while it is not reporting any new data.

In UDP, the manager realizes an agent is disconnected when 30 minutes go by without receiving any messages from that agent.

To sum up, an agent without reporting data to the manager in more than 30 minutes is marked as disconnected from the manager without any reason.

### Proposal solution

Create a new thread to send the keep-alive message every `notify_time` period, emulating the Linux agent behavior.

**Comment by crolopez** [comment]
| Development branch |
|---|
| 3963-windows-keepalive |

To solve this issue, we will remove the `send_win32_info` function and part of `SendMSG` for Windows because they are duplicating the functionality of the `run_notify` function and `start_agent`.

Instead of creating a sender thread, we will call to the `run_notify` function, which is only for Linux and needs to be adapted, from the function where the Windows agent receives the manager messages (`receiver_thread`). This is the closest way to unify the behaviour with the non-Windows agent.

The last commit of the development branch (5cc025757cf60845349ba04a1b56f7a51d168cf2) is a draft.

**Comment by chemamartinez** [comment]
Merged #4077
